fabric-biomes-v1/src/main/java/net/fabricmc/fabric/api/biomes/v1/FabricBiomeBuilder.java

```diff
@@ -0,0 +1,321 @@
+package net.fabricmc.fabric.api.biomes.v1;
+
+import com.google.common.collect.Lists;
+import com.google.common.collect.Maps;
+import net.fabricmc.fabric.mixin.biome.BiomeEffectsAccessor;
+import net.fabricmc.fabric.mixin.biome.SpawnDensityAccessor;
+import net.fabricmc.fabric.mixin.biome.SpawnSettingsAccessor;
+import net.minecraft.entity.EntityType;
+import net.minecraft.entity.SpawnGroup;
+import net.minecraft.sound.BiomeAdditionsSound;
+import net.minecraft.sound.BiomeMoodSound;
+import net.minecraft.sound.MusicSound;
+import net.minecraft.sound.SoundEvent;
+import net.minecraft.world.biome.*;
+import net.minecraft.world.gen.GenerationStep;
+import net.minecraft.world.gen.carver.ConfiguredCarver;
+import net.minecraft.world.gen.feature.ConfiguredFeature;
+import net.minecraft.world.gen.feature.ConfiguredStructureFeature;
+import net.minecraft.world.gen.surfacebuilder.ConfiguredSurfaceBuilder;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.function.Supplier;
+
+public class FabricBiomeBuilder {
```

Comment:
JSON biomes are deserialized into the same class as built-in biomes, so there is no difference at this point anymore, since this class is constructed from a built Biome instance.

Reply:
Yes, they behave the same.